src/PVRIptvData.cpp

```diff
@@ -804,39 +804,74 @@ int PVRIptvData::GetFileContents(std::string& url, std::string &strContent)
   return strContent.length();
 }
 
+/* 
```

Comment:
The comments are larger than the code added. Might be better to just credit the author and source. I.e.:

```
/*
  Adapted from https://stackoverflow.com/a/31533119
  Conversion from UTC date to second, signed 64-bit adjustable epoch version.
  Written by François Grieu, 2015-07-21; public domain. 
  */
```